cie_infra/__manifest__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "cie_infra",
```

Comment:
Name of the module can contains spaces and fancy characters. So you could name it : "Coop IT Easy Infrastructure".